extension/intents/music/music.js

```diff
@@ -115,5 +115,18 @@ intentRunner.registerIntent({
   },
 });
 
+intentRunner.registerIntent({
+  name: "music.find",
```

Comment:
`music.find` doesn't seem like a good description, it would make me think of searching or focusing a tab. `music.showTitle` would be clearer.